lib/rubocop/cop/layout/space_inside_reference_brackets.rb

```diff
@@ -110,8 +110,14 @@ def bracket_method?(node)
           BRACKET_METHODS.include?(method)
         end
 
-        def left_ref_bracket(tokens)
-          tokens.reverse.find(&:left_ref_bracket?)
+        def left_ref_bracket(node, tokens)
+          _, method = *node
+
+          if method.to_s == '[]='
```

Comment:
Does `method` need to be converted to a string? Would a comparison to a symbol work here instead? `method == :[]=`

Reply:
I tried it initially just comparing the symbols but it was not happy. I had to use a String comparison.

Reply:
The issue that you ran into with the debugger is `byebug` implements a `method` command.

Reply:
@rrosenblum Thanks for the clarification! That'll help my future self when I run into something like this again. 😀